packages/internal/src/__tests__/build_api.test.ts

```diff
@@ -93,9 +95,195 @@ test.skip('api prebuild transforms gql with `babel-plugin-graphql-tag`', () => {
+  expect(code).toContain(
+    `import _Reflect$getOwnMetadata from "@babel/runtime-corejs3/core-js/reflect/get-own-metadata";`
+  )
+
+  expect(code).toContain(
+    `import _Set from "@babel/runtime-corejs3/core-js/set";`
+  )
+  const _Set = require('@babel/runtime-corejs3/core-js/set')
+  expect(_Set).toHaveProperty('addAll')
+  expect(_Set).toHaveProperty('deleteAll')
+  expect(_Set).toHaveProperty('difference')
+  expect(_Set).toHaveProperty('every')
+  expect(_Set).toHaveProperty('filter')
+  expect(_Set).toHaveProperty('find')
+  expect(_Set).toHaveProperty('from')
+  expect(_Set).toHaveProperty('intersection')
+  expect(_Set).toHaveProperty('isDisjointFrom')
+  expect(_Set).toHaveProperty('isSubsetOf')
+  expect(_Set).toHaveProperty('isSupersetOf')
+  expect(_Set).toHaveProperty('join')
+  expect(_Set).toHaveProperty('map')
+  expect(_Set).toHaveProperty('of')
+  expect(_Set).toHaveProperty('reduce')
+  expect(_Set).toHaveProperty('some')
+  expect(_Set).toHaveProperty('symmetricDifference')
+  expect(_Set).toHaveProperty('union')
+
+  expect(code).toContain(
+    `import _codePointsInstanceProperty from "@babel/runtime-corejs3/core-js/instance/code-points";`
+  )
+  expect(code).toContain(
+    `import _matchAllInstanceProperty from "@babel/runtime-corejs3/core-js/instance/match-all";`
+  )
+  expect(code).toContain(
+    `import _replaceAllInstanceProperty from "@babel/runtime-corejs3/core-js/instance/replace-all";`
+  )
+  expect(code).toContain(
+    `import _atInstanceProperty from "@babel/runtime-corejs3/core-js/instance/at";`
+  )
+  expect(code).toContain(
+    `import _Symbol$patternMatch from "@babel/runtime-corejs3/core-js/symbol/pattern-match";`
+  )
+  expect(code).toContain(
+    `import _Symbol$dispose from "@babel/runtime-corejs3/core-js/symbol/dispose";`
+  )
+
+  expect(code).toContain(
+    `import _WeakMap from "@babel/runtime-corejs3/core-js/weak-map";`
+  )
+  const _WeakMap = require('@babel/runtime-corejs3/core-js/weak-map')
+  expect(_WeakMap).toHaveProperty('deleteAll')
+  expect(_WeakMap).toHaveProperty('from')
+  expect(_WeakMap).toHaveProperty('of')
+
+  expect(code).toContain(
+    `import _WeakSet from "@babel/runtime-corejs3/core-js/weak-set";`
+  )
+  const _WeakSet = require('@babel/runtime-corejs3/core-js/weak-set')
+  expect(_WeakSet).toHaveProperty('addAll')
+  expect(_WeakSet).toHaveProperty('deleteAll')
+  expect(_WeakSet).toHaveProperty('from')
+  expect(_WeakSet).toHaveProperty('of')
+
+  // Expect these to remain unchanged.
+  expect(code).toContain(
+    [
+      `const buffer = new ArrayBuffer(8);`,
+      `const uint8 = new Uint8Array(buffer);`,
+      `uint8.set([1, 2, 3], 3);`,
+    ].join('\n')
+  )
+
+  expect(code).toContain(
+    [
+      `[1, 2, 3].lastItem // => 3`,
+      `[(1, 2, 3)].lastIndex; // => 2`,
+      '',
+      `const array = [1, 2, 3];`,
+      `array.lastItem = 4;`,
+      `new Array(1, 2, 3).lastItem;`,
+      `new Array(1, 2, 3).lastIndex;`,
+    ].join('\n')
   )
```

Comment:
Even though these polyfills come up on the compat list, they don't work for whatever reason, so it's important to keep track of that too.

Reply:
It seems these ones are overriden by core-js-pure to have no implementation. See 
- https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override/modules/es.typed-array.set.js
- https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override/modules/esnext.array.last-index.js
- https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override/modules/esnext.array.last-item.js

Also from their contributing docs:

> If the code of the pure version implementation should significantly differ from the global version (that's not a frequent situation, in most cases internals/is-pure is enough), you can add it to [packages/core-js-pure/override](https://github.com/zloirock/core-js/blob/master/packages/core-js-pure/override) directory. The rest parts of core-js-pure will be copied from core-js package.

Src: https://github.com/zloirock/core-js/blob/master/CONTRIBUTING.md#how-to-add-a-new-polyfill